starter-code/index.html

```diff
@@ -2,30 +2,40 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" type="text/css" href="./css/main.css">
 </head>
-<body>
-  Instagram
 
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
-
-  &copy; 2014 Instagram
+<body class="container">
+<div class="clr"></div>
+  <div class="phone-img">
+    <img src="./images/phones.png" alt="phones"/>
+  </div>
+  <div>
+    <div>
+        <img src="./images/brand.png" alt="instagram logo"/>
+        <button type="button" onclick="alert('Thank you for testing')"><img src="./images/home.png"/>Log in</button>
+    </div>
+    <div>
+      <h3>Capture and Share the World's Moments</h3>
+      <p>Instagram is a <b>fast</b>,<b>beautiful</b> and <b>fun</b> way to share your life with friends and family.</p>       
+      <p>Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
+      <p>Oh yeah, did we mention it's free?</p> 
+    </div>
+    <div>
+      <img src="./images/badge-iphone.png" alt="Download on the App Store"/>
+      <img src="./images/badge-android.png" alt="  Get it on Google Play"/>
+    </div> 
+  </div>
+  <footer>
```

Comment:
the footer should be at the bottom of clone and be set like link. ( you can set an empty link by doing ` <a href="#"> Blah </a>`